rtsp.go

```diff
@@ -727,7 +678,36 @@ func (rc *rtspCamera) initH265(session *description.Session) (err error) {
 		} else {
 			rc.storeH265Frame(packedAU)
 		}
-	})
+	}
+
+	onPacketRTP := func(pkt *rtp.Packet) {
+		// we need to do this before calling Decode as apparently rtpDec.Decode mutates
+		// the packet such that one can't get the pts out after Decode is called
+		pts, ok := rc.client.PacketPTS2(media, pkt)
+		if !ok {
+			rc.logger.Debug("no pts found for packet")
+			return
+		}
+
+		// Extract access units from RTP packets
+		au, err := rtpDec.Decode(pkt)
+		if err != nil {
+			if !errors.Is(err, rtph265.ErrNonStartingPacketAndNoPrevious) && !errors.Is(err, rtph265.ErrMorePacketsNeeded) {
+				rc.logger.Debugw("error decoding(1) h265 rstp stream", "err", err.Error())
+			}
+			return
+		}
+		storeImage(au)
+		rc.videoStoreMuxer.writeH265(au, pts)
```

Comment:
[q] Still a little unclear about what the behavior of the rawsegmenter in the case that first AU write here is not a keyframe.

Reply:
we won't call it without a keyframe